Alert/Alert.css

```diff
@@ -0,0 +1,134 @@
+@import "../../utility/component.css";
+
+.alert{
+    position: relative;
+    padding: 1rem 1rem;
+    margin-bottom: 1rem;
+    border: 1px solid transparent;
+    border-radius: 0.25rem;
+}
+
+/*---alert example---*/
+
+#show_alert{
+    display: none;
+}
+
+#alert_example{
+    display: none;
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    z-index: 999;
+    transform: translateX(100%);
+    -webkit-transform: translateX(100%);
+    animation: slide-in 0.5s forwards;
+    -webkit-animation: slide-in 0.5s forwards;
+}
+
+/*----alert-close-icon---*/
+
+.close_alert{
+  margin-left: 0.938rem;
+  color: white;
+  font-weight: bold;
+  float: right;
+  font-size: 1.375rem;
+  line-height: 1.25rem;
+  cursor: pointer;
+  z-index: 1;
+  transition: 0.3s;
+}
+
+/*----action-alert---*/
+
+.action_alert{
+  display: block;
+  display: flex;
+  justify-content: space-between;
+}
+
+  @keyframes slide-in {
+    0% {
+      -webkit-transform: translateX(100%);
+    }
+    100% {
+      -webkit-transform: translateX(0%);
+    }
+  }
+  
+  @-webkit-keyframes slide-in {
+    0% {
+      transform: translateX(100%);
+    }
+    100% {
+      transform: translateX(0%);
+    }
+  }
+  
+
+/*--------outline---border----*/
+
+
+.alert-outline-success{
+    border : 1px solid var(--green);
```

Comment:
I think, the purpose of using variables is we can change colors as per our needs. If we name them like this then no point of using variable. So, better do naming which doesn't block it's future possibilities of changes   like here you can create and use var(--alert-success-border)